Web/ForgeTrust.Runnable.Web.RazorWire/wwwroot/razorwire/razorwire.js

```
@@ -315,15 +317,148 @@
+                this.updateInterval = null;
+            }
+        }
+
+        handleMutations(mutations) {
+            for (const mutation of mutations) {
+                for (const node of mutation.addedNodes) {
+                    if (node instanceof Element) {
+                        if (node.hasAttribute('data-rw-time')) {
+                            this.format(node);
+                        }
+                        node.querySelectorAll('time[data-rw-time]').forEach(el => this.format(el));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)